lib/oli/delivery/sections.ex

```diff
@@ -5349,4 +5350,32 @@ defmodule Oli.Delivery.Sections do
     )
     |> Repo.all()
   end
+
+  def get_section_prompt_info(section_id) do
+    %Section{customizations: customizations} = section = Repo.get(Section, section_id)
+
+    instructors =
+      section.slug
+      |> fetch_instructors()
+      |> Enum.map(&%{name: &1.name, email: &1.email})
+
+    layout =
+      section_id
+      |> SectionResourceDepot.get_section_resources_by_type_ids([ResourceType.id_for_container()])
```

Comment:
For consistency sake, make one call to SectionResourceDepot.get_section_resources_by_type_ids for `[ResourceType.id_for_container(), ResourceType.id_for_page()]` then post process to separate the two types. 

Also, `Oli.Delivery.Sections` should not depend on or use `SectionResourceDepot`.   The dependency is the other way around.  So instead, move this entire function out of `Oli.Delivery.Sections` into a different contenxt, perhaps something `Converstation` specific.

Reply:
👍 Fixed in f57ed5c4034806a6a70701b2c77afe0526850e50
Made one call to `SectionResourceDepot.get_section_resources_by_type_ids/2` and  moved the function to the `Oli.Conversation.Functions` module